### PR TITLE
[Doctrine] Slave/Master configuration options

### PR DESCRIPTION
**Pull request by Simperfit**
Like settled in the [Configuration Class](https://github.com/doctrine/DoctrineBundle/blob/master/DependencyInjection/Configuration.php#L149) options keep_slave should be a children of connections

edit : preview : http://pr-7111-scvfh6y-6qmocelev2lwe.eu.platform.sh/reference/configuration/doctrine.html

### CODE REVIEW

File: reference/configuration/doctrine.rst [hunk 1]
```diff
@@ -406,7 +408,6 @@ The following block shows all possible configuration keys:
                 types:
                     custom: Acme\HelloBundle\MyCustomType
                 # the DBAL keepSlave option
```
Comment (xabbuh) [inline reference/configuration/doctrine.rst]: the comment must be moved too

**Review by xabbuh — CHANGES_REQUESTED** [review]

**Comment by Simperfit** [comment]
@xabbuh comment taken in account

**Comment by javiereguiluz** [comment]
👍

**Comment by xabbuh** [comment]
Thank you @Simperfit.

**Pull request closed without merging**